logicmonitor_data_sdk/configuration.py

```diff
@@ -79,23 +82,37 @@ def __init__(self, company=None, authentication=None, id=None, key=None):
       # type = os.environ.get('LM_ACCESS_TYPE', 'LMv1')
```

Comment:
Do we have env for this bearer token?

Reply:
@lmsohamdeshmukh Can you please respod?

Reply:
Made changes to incorporate LM_BEARER_TOKEN env variable and removed LMV1authflag